Tools/Scripts/libraries/webkitscmpy/webkitscmpy/remote/git_hub.py

```diff
@@ -649,3 +641,27 @@ def find(self, argument, include_log=True, include_identifier=True):
         if not commit_data:
             raise ValueError("'{}' is not an argument recognized by git".format(argument))
         return self.commit(hash=commit_data['sha'], include_log=include_log, include_identifier=include_identifier)
+
+    def create(self, path: str, data: dict, headers: dict = None) -> Optional[requests.Response]:
```

Comment:
Remove `Optional` from typing hint. This will always return `requests.Response`.

Reply:
We are still supporting Python 2 in some of the consumers of this library. We need to discuss deprecating Python 2 for those use cases, but that's why our checks still run Python 2 unit tests.

Reply:
Ah I see. In that case are we better off removing typing hints?

Reply:
Yes, that's basically why we don't use type hints in these libraries at the moment